### PR TITLE
  Feature/from old model

### PR DESCRIPTION
**Pull request by l4l**
### Description of the Change

Make new shared model with older ones

### Benefits

Easier migration

### CODE REVIEW

File: shared_model/backend/protobuf/from_old_model.hpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+#include "backend/protobuf/block.hpp"
+#include "backend/protobuf/proposal.hpp"
+#include "backend/protobuf/queries/proto_query.hpp"
+#include "backend/protobuf/transaction.hpp"
+#include "model/coverters/pb_block_factory.hpp"
+#include "model/coverters/pb_query_factory.hpp"
+#include "model/coverters/pb_transaction_factory.hpp"
+
+inline static shared_model::proto::Block from_old(
```
Comment (nickaleks) [inline shared_model/backend/protobuf/from_old_model.hpp]: Is there a reason why functions are declared static?

File: shared_model/backend/protobuf/from_old_model.hpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    const iroha::model::Query &qry) {
+  return shared_model::proto::Query(PbQueryFactory().serialize(qry));
+}
+
+inline static shared_model::proto::Proposal from_old(
+    const iroha::model::Proposal &proposal) {
+  iroha::protocol::Proposal proto;
+  proto.set_height(proposal.height);
+  for (const auto &tx : proposal.transactions) {
+    new (proto.add_transactions())
```
Comment (muratovv) [inline shared_model/backend/protobuf/from_old_model.hpp]: Maybe better to use `mutable` interface instead.

**Comment by nickaleks** [comment]
Why are all the functions in global namespace?

File: shared_model/backend/protobuf/from_old_model.hpp [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        protocol::Transaction(factory_.serialize(tx));
+  }
+  return shared_model::proto::Proposal(proto);
+}
+
+#endif  // DISABLE_BACKWARD
```
Comment (muratovv) [inline shared_model/backend/protobuf/from_old_model.hpp]: Pls, add the namespace.

**Pull request merged**